### PR TITLE
tssk9

### PR DESCRIPTION
**Pull request by Rue-ops6**

### CODE REVIEW

File: app/Http/Controllers/ProdController.php [hunk 1]
```diff
@@ -11,16 +11,63 @@ class ProdController extends Controller
...
+     * Display a listing of the resource.
+     */
+    public function LatestProds()
+    {
+        /* 
+        #SELECT * FROM `products` ORDER BY updated_at DESC limit 3;
+        */
+        #$products = Product::orderBy('updated_at', 'desc')->limit(3)->get();
+        $prods = Product::where('pub', 1)
+        ->orderBy('updated_at', 'DESC')
```
Comment (Nora-Lootfy) [inline app/Http/Controllers/ProdController.php]: you can replace `orderBy('updated_at', 'DESC')` by `latest()`

File: public/assets/images/1722965893.jpg
Comment (Nora-Lootfy) [inline public/assets/images/1722965893.jpg:1]: nice😂

**Review by Nora-Lootfy — APPROVED** [review]
so proud😊

File: public/assets/images/1722965893.jpg
Comment (Rue-ops6) [reply public/assets/images/1722965893.jpg:1]: laaaa mtrag3i4 waraia bdmirrrr keda X""""""""D

File: app/Http/Controllers/ProdController.php (hunk 1 above)
Comment (Rue-ops6) [reply app/Http/Controllers/ProdController.php]: ok 4okran mo5tasara el 2 fe3ln

**Pull request merged**